vite.config.ts

```diff
@@ -0,0 +1,22 @@
+import { resolve } from 'path';
```

Comment:
Can we try using `node:` prefix?

```suggestion
import { resolve } from 'node:path';
```

Reply:
sure